ql/src/java/org/apache/hadoop/hive/ql/parse/TransformSpec.java

```diff
@@ -76,6 +90,28 @@ public String transformTypeString() {
     return transformType.name();
   }
 
+  public String toHiveExpr(Configuration conf, boolean alias) {
```

Comment:
this method looks testable, also it's a string magic which is hard to read and understand, I would love to see a unit test with some test cases with input confs -> output string pairs